src/internal/constants/dropdowns.ts

```diff
@@ -11,6 +11,18 @@ export const pushpinSizeDropdownItems: IDropdownItem[] = [
     { title: "properties.points.size.extra_large", value: "1.5x" },
 ];
 
+export const pushpinViewportDropdownItems: IDropdownItem[] = [
+    { title: "properties.viewport.area.auto_default", value: "auto" },
+    { title: "properties.viewport.area.world", value: "world" },
```

Comment:
should it have separator here?
{ type: "separator" },

Reply:
I replaced `{ type: "separator" } `  with `{ type: "header", title: "properties.viewport.area.continents" }`